python/lsst/ts/aos/utils/scripts/lut_update.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import asyncio
+from astropy.time import Time
+
+import numpy as np
+from numpy.polynomial import Polynomial
+import os
+import pandas as pd
+from tqdm import tqdm
+
+from lsst_efd_client import EfdClient
+from lsst.ts.cRIOpy import M1M3FATable
+
+async def update_lut_force_balance(start_time, end_time, axis, lut_path, polynomial_degree, resample_rate) -> None:
+    """ Update the LUT file with the force balance data from the EFD. 
+    Saves a new .csv file with LUT updated values
+
+    Parameters
+    ----------
+    start_time: astropy.time.Time
+        Start time of the data to be queried from the EFD
+    end_time: astropy.time.Time
+        End time of the data to be queried from the EFD
+    axis: str
+        Axis of the force balance to be updated. Can be 'X', 'Y', or 'Z'
+    lut_path: str
+        Path to the LUT file to be updated
+    polynomial_degree: int
+        Degree of the polynomial to be fitted to the data
+    resample_rate: str
+        Rate at which the data is resampled. Default is '1T' (1 minute)
+
+    Returns
+    -------
+    None
+    """
+    
+    # Initialize EFD client
+    client = EfdClient('usdf_efd')
+
+    # Get the number of actuators
+    print('1')
+    num_actuators =  M1M3FATable.FATABLE_ZFA
+    if axis == 'X':
+        num_actuators = M1M3FATable.FATABLE_XFA
+    elif axis == 'Y':
+        num_actuators = M1M3FATable.FATABLE_YFA
+    print('2')
+
+    # Get path of the LUT file
+    lut_file = os.path.join(lut_path, f"Elevation{axis}Table.csv")
+    table_file = pd.read_csv(lut_file)
+    print('3')
+    # Generate array with the ID of each actuator
+    fat = np.array(M1M3FATable.FATABLE, dtype=object)
+    ids = fat[:, M1M3FATable.FATABLE_ID]
+
+    # Get names of actuator forces
+    query_forces = [f"{axis.lower()}Forces{i}" for i in range(num_actuators)]
+
+    # Query appliedBalanceForces from EFD from start_time to end_time
+    forces = await client.select_time_series(
```

Comment:
Please add some human-friendly output when appliedBalanceForces will be empty/low in numbers and will not cover all elevations.